### DISCUSSION TITLE
vc-imagery-provider-wmts 顶部为什么有一个圆形

### DISCUSSION THREAD

**Discussion by xllyll** [zouyaoji/vue-cesium#517] [Q&A]
@
![微信截图_20240220145339](https://github.com/zouyaoji/vue-cesium/assets/7789193/495a12c5-a25c-43bc-aa6d-acdb2a338130)
怎么可以去掉

**Comment by zouyaoji** [comment]
这是坐标系限制哈，示例加载的是3857的数据，你加4326的应该没这问题。
![image](https://github.com/zouyaoji/vue-cesium/assets/24430116/f170f826-e8de-4539-baae-8d2b4167987f)

**Reply by xllyll** [reply]
我直接用cesium.js代码加载我们的url 没有上面的圆
![微信截图_20240226111356](https://github.com/zouyaoji/vue-cesium/assets/7789193/1df569f9-cd7b-4896-9452-e51e31930518)
换成vue-cesium
![微信截图_20240226111426](https://github.com/zouyaoji/vue-cesium/assets/7789193/d0081b9f-af60-4e9b-9264-3a7d49d8cf0b)
就会出现顶部的圆

**Reply by zouyaoji** [reply]
官方的也有啊。
![image](https://github.com/zouyaoji/vue-cesium/assets/24430116/c846f495-2111-4b63-b35b-c29acf938612)
如果你的没有 只能说明你加的是4326的数据，确保用vue-cesium组件加的也是4326

**Reply by xllyll** [reply]
我确认就是4326的数据，怎么可以处理呢？

**Reply by zouyaoji** [reply]
有没有全球的4326的wmts在线服务发一下，我找了一圈没看到有，没法尝试。。。

**Reply by zhoubonet** [reply]
坐标系：4490也是这样
![Uploading image.png…]()

**Reply by zhoubonet** [reply]
<img width="663" alt="image" src="https://github.com/user-attachments/assets/c195dd8b-664e-4679-a76f-30b2fd3936c0">
坐标系：4490也是这样

**Reply by zhoubonet** [reply]
<img width="754" alt="image" src="https://github.com/user-attachments/assets/3e41db90-c70a-4417-8598-8608100f66a0">
配置信息
